dstyle.dd

```diff
@@ -421,9 +421,9 @@ bool isPositive(int number)
 }
 ---
     $(LI Text in sections (e.g. `Params:`, `Returns:`, `See_Also`) should be indented by one level if it spans more than the line of the section.)
-    $(LI Documentation comments should not use more than two stars `/**` in the header line.)
-    $(LI Block comments (`/**`) should be used - not nesting block comments (`/++`))
-    $(LI Global functions shouldn't indent their documentation block nor use stars as indentation.)
+    $(LI Documentation comments should not use more than two stars `/**` or two pluses `/++` in the header line.)
+    $(LI Either Block comments (`/**`) or nesting block comments (`/++`) should be used except when the ddoc comment is a ditto comment such as `/// Ditto`)
```

Comment:
Now the next question would be: do we recommend `/// Ditto` or `/// ditto` for new code?
Stats on Phobos are 575x on `ditto` and 496x on `Ditto`

It's obviously a subjective choice again.

Reply:
I don't see much reason to care. I know that I'm completely inconsistent with regards to which I use, and sometimes I forget and wonder if I have to use ditto or Ditto or whether they both work, but a quick check of Phobos shows that they both work. I'd just as soon not require one or the other. Certainly, if one were required, I don't expect that I'd ever remember which, and I'd have to check every time.

Reply:
I only added mention of Ditto to the style guide so that we're allowed to use `///` with it.

Reply:
Oh the reason was rather simple: whatever is written as example on this style guide will likely be the way newcomers write their code or other projects choose to base their style guide on (by saying "Ditto" and not "ditto" is passively endorsed.)
So if you or someone else has a preference between "ditto" and "Ditto" - now is the time to speak.
I don't care much about this either.